src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -426,6 +453,7 @@
                     "Name": "SecurityGroups",
                     "Description": "A list of IDs of security groups that App Runner should use for access to AWS resources under the specified subnets. If not specified, App Runner uses the default security group of the Amazon VPC. The default security group allows all outbound traffic.",
                     "Type": "List",
+                    "TypeHint": "ExistingSecurityGroups",
```

Comment:
Set the `ParentSettingId` equal to `VpcId`.

Reply:
Done